pkg/controller/utils/metadata/operator_metadata.go

```diff
@@ -79,21 +74,11 @@ func NewOperatorMetadataForwarder(logger logr.Logger, k8sClient client.Reader, k
 
 // Start starts the operator metadata forwarder
 func (omf *OperatorMetadataForwarder) Start() {
-	err := omf.setCredentials()
-	if err != nil {
-		omf.logger.Error(err, "Could not set credentials; not starting operator metadata forwarder")
-		return
-	}
-
 	if omf.hostName == "" {
-		omf.logger.Error(ErrEmptyHostName, "Could not set host name; not starting operator metadata forwarder")
+		omf.logger.Error(ErrEmptyHostName, "Could not set host name; not starting helm metadata forwarder")
 		return
 	}
 
-	omf.payloadHeader = omf.getHeaders()
-
-	omf.updateResourceCounts()
```

Comment:
```suggestion
	omf.updateResourceCounts()
```
add this back, so any payloads sent at the start of the metadata forwarder include the latest resource counts